index.rst

```diff
@@ -1,6 +1,8 @@
-#################################
-XMOS Avona Voice Reference Design
-#################################
+################
+Voice Framework
```

Comment:
The README.rst file uses the title 'XMOS Voice Framework DSP Library'.  I do not see a need to use exactly the same name everywhere, but it does make me wonder if this line should be 'XMOS Voice Framework'.

Reply:
I referred to some other framework libraries and they weren't using XMOS either. It was IO Framework, Core Framework etc. so I think just Voice Framework follows the convention. Also, it is the xmos/fwk_voice repo so don't think specifying XMOS as part of the name is necessary. Let me know if you want me to change it. Thanks